ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/advanced-gas-fee-input-subtext.js

```diff
@@ -1,42 +1,70 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { isNullish } from '../../../../helpers/utils/util';
+import { formatGasFee } from '../../../../helpers/utils/gas';
 import Box from '../../../ui/box';
 import I18nValue from '../../../ui/i18n-value';
 import LoadingHeartBeat from '../../../ui/loading-heartbeat';
 
-const AdvancedGasFeeInputSubtext = ({ latest, historical, feeTrend }) => {
```

Comment:
I ended up changing this component slightly. This component no longer takes preformatted strings for latest fee and historical fee (e.g. "123 GWEI" or "1.4 - 3 GWEI"), but the direct fee or fee range (e.g `'123.2345'` or `['1.423432', '3.2343']`). I introduced a new utility function, `formatGasFee`, which this component uses. This function returns something like a) "123 GWEI" if a singular value is given, b) "1.4 - 3 GWEI" if a range is given, or c) null if a null or undefined is given. This function is used in other places as well.